src/components/pages/Login.js

```diff
@@ -1,11 +1,75 @@
-import React from 'react';
+import React from "react";
+import { useState } from "react";
+import { Redirect } from "react-router-dom";
+import axios from "axios";
+
+function Login(props) {
+  const [credentials, changeCredentials] = useState({
+    email: "",
+    password: ""
+  });
+
+  const [status, changeStatus] = useState({
+    loading: false,
+    error: false
+  });
+
+  const submitLogin = event => {
+    event.preventDefault();
+    changeStatus({ ...status, loading: true });
+    axios
+      .put("https://tieme-ndo-backend-production.herokuapp.com/", {
+        email: credentials.email,
+        password: credentials.password
+      })
+      .then(res => {
+        if (res.token) {
+          localStorage.setItem("token", res.token);
```

Comment:
All this localStorage thing will be performed outside of the Login page, probably in a hoc cc @AnthonyJCampbell

Reply:
Yes, whenever that is finished, see above 👍